client/src/www/app/outline_server_repository/vpn.ts

```diff
@@ -98,4 +98,7 @@ export interface VpnApi {
 
   /** Sets a listener, to be called when the tunnel status changes. */
   onStatusChange(listener: (id: string, status: TunnelStatus) => void): void;
+
+  /** Fetches the config from a dynamic key URL. */
+  fetchDynamicConfig(url: string): Promise<string>;
```

Comment:
This is not part of the VPN API. Please remove. We need to put it somewhere else.

Reply:
How about putting it in nativeHelper?

Reply:
We already have the concept of `OutlinePlatform`: https://github.com/Jigsaw-Code/outline-apps/blob/master/client/src/www/app/platform.ts

I think we need to add something like a `getUrlFetcher()`?

Reply:
Done, added `getResourceFetcher`.